### DISCUSSION TITLE
This site can’t be reached

### DISCUSSION THREAD

**Discussion by CROprogrammer** [microcks/microcks#708] [Q&A]
After executing 
`helm install microcks ./microcks --namespace=microcks^
 --set appName=microcks --set features.async.enabled=true^
 --set microcks.url=microcks.192.168.49.2.nip.io^
 --set keycloak.url=keycloak.192.168.49.2.nip.io^
 --set features.async.kafka.url=192.168.49.2.nip.io`

I get message:

NAME: microcks
LAST DEPLOYED: Mon Nov 28 14:19:31 2022
NAMESPACE: microcks
STATUS: deployed
REVISION: 1
TEST SUITE: None
NOTES:
Thank you for installing microcks.

Your release is named microcks.

To learn more about the release, try:

  $ helm status microcks
  $ helm get microcks

Microcks is available at https://microcks.192.168.49.2.nip.io.

GRPC mock service is available at "microcks-grpc.192.168.49.2.nip.io".
It has been exposed using TLS passthrough on the Ingress controller, you should extract the certificate for your client using:

  $ kubectl get secret microcks-microcks-grpc-secret -n microcks -o jsonpath='{.data.tls\.crt}' | base64 -d > tls.crt

Keycloak has been deployed on https://keycloak.192.168.49.2.nip.io/auth to protect user access.
You may want to configure an Identity Provider or add some users for your Microcks installation by login in using the
username and password found into 'microcks-keycloak-admin' secret.

Kafka broker has been deployed on microcks-kafka.192.168.49.2.nip.io.
It has been exposed using TLS passthrough on the Ingress controller, you should extract the certificate for your client using:

  $ kubectl get secret microcks-kafka-cluster-ca-cert -n microcks -o jsonpath='{.data.ca\.crt}' | base64 -d > ca.crt

And after that I'm trying to go to https://keycloak.192.168.49.2.nip.io/auth and I get error in browser "This site can’t be reached". This is the case with any other link i get in that message.
Everything is green in kubernetes.
Any ideas why this keeps happening?

**Comment by CROprogrammer** [comment]
![image](https://user-images.githubusercontent.com/73464772/204767253-73fc09b2-62f4-4512-8f72-476c253df45a.png)

**Reply by lbroudoux** [reply]
Looks like an issue on how your minikube VM IP is reached from your host system ... Have you checked if other apps Ingresses are working correctly?

**Reply by CROprogrammer** [reply]
I've exposed it on another url and I seem to be able to access KeyCloak, but now I have a problem where I'm trying to sign in into KeyCloak with credientials written in microcks-keycloak-admin secret, but it does not work for some reason

**Reply by CROprogrammer** [reply]
> Looks like an issue on how your minikube VM IP is reached from your host system ... Have you checked if other apps Ingresses are working correctly?

I tried deploying another application and it worked fine

**Reply by lbroudoux** [reply]
Hey @michelh-fer 

I just had the opportunity to re-deploy everything from scratch on a new cluster and things are working well when ingresses are correctly setup (lately with the number of new drivers and new arm64 arch, it has became a real pain with minikube ...)

I also checked authentication to Keycloak admin works well. You have to extract user and password from secret using those commands:

```sh
$ kubectl get secret microcks-keycloak-admin -n microcks -o jsonpath='{.data.username}' | base64 -d 
$ kubectl get secret microcks-keycloak-admin -n microcks -o jsonpath='{.data.password}' | base64 -d
```

**Reply by CROprogrammer** [reply]
I tried getting username and password like that and it still wouldn't let me through

Here is the list of commands I'm executing:

minikube start

kubectl patch -n ingress-nginx deployment/ingress-nginx-controller --type=json -p [{"""op""":"""add""","""path""":"""/spec/template/spec/containers/0/args/-""","""value""":"""--enable-ssl-passthrough"""}]

kubectl create namespace microcks

helm repo add strimzi https://strimzi.io/charts/
helm install strimzi strimzi/strimzi-kafka-operator --namespace microcks

helm install microcks ./microcks --namespace=microcks^
 --set appName=microcks --set features.async.enabled=true^
 --set microcks.url=microcks.192.168.49.2.nip.io^
 --set keycloak.url=keycloak.192.168.49.2.nip.io^
 --set features.async.kafka.url=192.168.49.2.nip.io

After those I still couldn't access microcks or keycloak urls from console so I executed: minikube service --all -n microcks --url
And after that I could access Microcks in browser (but all it showed was blank white page) and I could access Keycloak page, but I could not log in with username and password from keycloak secret
